application/src/main/java/org/togetherjava/tjbot/commands/bookmarks/BookmarksSystem.java

```diff
@@ -39,7 +39,7 @@ public final class BookmarksSystem {
     static final Color COLOR_FAILURE = new Color(238, 153, 160);
 
     private final Database database;
-    private final Predicate<String> isOverviewChannelName;
+    private final Predicate<String> isHelpForumName;
```

Comment:
Usually when variables are named `isXXX` it means the variable is a boolean. Should we maybe change the naming here to make it more clear it's not?

Reply:
What would you advise then? Reason this is named like this is because a Predicate returns a boolean.
The predicate checks or it "is the help forum name", which is why we use this naming

Reply:
A predicate essentially represents a method. You would also call the method `isFoo()`, so it makes sense (and afaik is idiomatic) to also name the predicate like this.

`isFoo.test(x)`